emailtemplates/email.py

```diff
@@ -119,6 +119,11 @@ def render_message(self):
         self.message = message
 
     def get_message_object(self, send_to, attachment_paths, *args, **kwargs):
+        if kwargs.get('reply_to') is None:
```

Comment:
Your code is good, but with one amendment:
`if defaut_reply_to_email:` => `reply_to and type(reply_to) == str`

Why? because you can pass empty list in argument to remove default reply-to e-mail.